qiskit/transpiler/preset_passmanagers/generate_preset_pass_manager.py

```diff
@@ -331,7 +332,16 @@ def generate_preset_pass_manager(
         if timing_constraints is None:
             timing_constraints = target.timing_constraints()
         if backend_properties is None:
```

Comment:
same comment as in transpile, shoudln't we warn users about the `backend` input deprecating `BackendV1` as an option?

Reply:
Adding it in 048fef553903b8c0097276125887d14ebb3a2788